.env.dist

```diff
@@ -1,6 +1,13 @@
 #DEFAULT_LOCALE=en
 #SERVICE_NAME=articles
 
+###> doctrine/doctrine-bundle ###
+# Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
+# For an SQLite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
+# Configure your db driver and server_version in config/packages/doctrine.yaml
+DATABASE_URL=mysql://db_user:db_password@127.0.0.1:3306/db_name
```

Comment:
Symfony Flex's default, could tweak to say `pgsql`.

Reply:
Would be consistent with the containers used in the development environment

Reply:
Also hostname, port, username, password, database name...

Reply:
https://github.com/libero/content-store/blob/master/.travis.yml#L11 will probably need a `ln -s .env.dist .env` to be executed before the build

Reply:
Ah this is not for `docker-compose`... I think

Reply:
No, it's for Symfony itself (so the values you should put into `docker-compose`, rather than values themselves).